signal_id/_moment_errs.py

```diff
@@ -175,7 +175,7 @@ def moment2_error(cube, scale, axis, how='auto', moment0=None, moment1=None,
 
     # Remove velocity offset to match cube._pix_cen
     # Requires converting to a Quantity
-    moment1 = u.Quantity(moment1)
+    moment1 = u.Quantity(moment1.copy())
```

Comment:
making it a Quantity doesn't automatically copy the data?  I guess I didn't realize that.  It might be better to have `copy=True` as a parameter, since that's default for, e.g., `np.array`